install-ruby/README.md

```diff
@@ -2,25 +2,30 @@
 
 You'll either need to specify `ruby-version` or `ruby-version-file`
 
+## With a .ruby-version file
+
+If your project has a `.ruby-version` file:
 
 ```yaml
 tasks:
   - key: ruby
-    call: mint/install-ruby 1.0.1
+    call: mint/install-ruby 1.0.2
     with:
-      ruby-version: 3.2.2
+      ruby-version-file: .ruby-version
+    filter: [.ruby-version]
 ```
 
-Or with a file named `.ruby-version` containing the version of ruby to install:
+Remember to include the [`filter`](https://www.rwx.com/docs/mint/filtering-files) so that the task will be cached only based on the contents of the `.ruby-version` file.
```

Comment:
We had talked about just moving that filter into the definition of this leaf — any reason not to do that? The leaf never needs any other files so it presumably could have the first task have filter: [] and the second task filter:[${{ params.version-file }}]]

Reply:
I'll give it a shot. Is there any advantage to pulling the filter up to the leaf as a whole? Can we produce a cache hit at that level rather than having to cache hit on the tasks within the leaves?

Reply:
I don't think we support expressions in `filter` yet:

https://cloud.rwx.com/mint/rwx/tasks/f4ba4481b0c06d4094a423a58a2ef118/diagnostics